frontend/src/App/AppUpdatedModalContent.js

```diff
@@ -76,7 +76,7 @@ function AppUpdatedModalContent(props) {
             <div>
               {
                 !update.changes &&
-                  <div className={styles.maintenance}>Maintenance release</div>
+                  <div className={styles.maintenance}>Maintenance Release: bug fixes and other improvements. See Github Commit History for more details</div>
```

Comment:
```suggestion
                  <div className={styles.maintenance}>Maintenance Release: Bug fixes and other improvements. See <a href="https://github.com/Sonarr/Sonarr" title="GitHub Project" target="_blank" rel="noopener">GitHub Commit History</a> for more details.</div>
```

Maybe?

Reply:
A link to GitHub commits seems reasonable, but should use Link.

Bug fixes should not be part of a build that only contains minor code changes, I think something like:

> Maintenance Release: See <GitHub Commits> for the technical details